apps/api-server/src/routes/widget/widget.js

```diff
@@ -0,0 +1,149 @@
+const express = require('express');
+const bruteForce = require('../../middleware/brute-force');
+const db = require('../../db');
+const fs = require('fs');
+const config = require('config');
+const path = require('path');
+
+let router = express.Router({ mergeParams: true });
+
+// brute force
+router.use(bruteForce.globalMiddleware);
+
+router
+  .route('/:widgetId([a-zA-Z0-9]+)')
+  .all((req, res, next) => {
+    if (!req.params?.widgetId) {
+      return next();
+    }
+
+    db.Widget.findOne({
+      where: { id: req.params.widgetId },
+      include: ['project'],
+    })
+      .then((found) => {
+        if (!found) throw new Error('Widget not found');
+        req.widget = found;
+        next();
+      })
+      .catch(next);
+  })
+  .get((req, res) => {
+    const widgetId = req.params.widgetId;
+    const randomId = Math.floor(Math.random() * 1000000);
+    const widget = req.widget;
+
+    // @todo: add all widgets
+    const widgetSettingsMapping = {
+      arguments: {
+        js: ['comments.js'],
+        css: ['css/comments.css'],
+        name: 'Comments',
+        defaultConfig: {
+          ideaId: '1',
```

Comment:
moet ideaId niet vervangen worden met widgetId?

Reply:
Goed punt! Dat heb ik nu toegevoegd, standaard heb ik 'em leeggelaten, dan moet het omvattende component (bijv `resource-representation` dit ID doorgeven). In de Comments component moeten we hier wel rekening mee houden — het ideaId kan dan ook leeg zijn.